src/checkout/NotificationSettingsForm.tsx

```diff
@@ -44,7 +44,7 @@ const NotificationSettingsForm: FC = () => {
                 label="Limit ($1 minimum)"
                 min={1}
                 required
-                step={0.01}
+                step={1}
```

Comment:
this was a design ask, apparently it was the original implementation that got changed